### PR TITLE
[BD-2] BB-2355: Add LTI AGS basic support

### PR DESCRIPTION
**Pull request by giovannicimolin**
TBD.

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @giovannicimolin! I've created [OSPR-4676](https://openedx.atlassian.net/browse/OSPR-4676) to keep track of it in JIRA, where we prioritize reviews.  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).

Please let us know once your PR is ready for our review and all tests are green.

File: lti_consumer/lti_consumer.py [hunk 1]
```diff
@@ -827,6 +828,109 @@ def _get_lti1p3_consumer(self):
...
+            return Response(status=404)
+
+        lti_consumer = self._get_lti1p3_consumer()
+        return Response(
+            json_body=self.ags_block_lineitem,
+            status=200,
+        )
+
+    @XBlock.handler
+    def lti_ags_results(self, request, suffix=''):  # pylint: disable=unused-argument
```
Comment (giovannicimolin) [inline lti_consumer/lti_consumer.py:923]: WIP

File: lti_consumer/lti_consumer.py [hunk 2]
```diff
@@ -827,6 +828,109 @@ def _get_lti1p3_consumer(self):
...
+            return Response(status=404)
+
+        return Response(
+            json_body=self.ags_block_lineitem,
+            status=200,
+            content_type="application/vnd.ims.lis.v2.lineitem+json",
+        )
+
+    @XBlock.handler
+    def lti_ags_scores(self, request, suffix=''):  # pylint: disable=unused-argument
```
Comment (giovannicimolin) [inline lti_consumer/lti_consumer.py:909]: WIP

**Comment by giovannicimolin** [comment]
@davestgermain @davidjoy FYI, wip version of read only LTI AGS.

File: lti_consumer/utils.py [hunk 3]
```diff
@@ -58,3 +58,26 @@ def get_lms_lti_access_token_link(location):
         lms_base=get_lms_base(),
         location=text_type(location),
     )
+
+
+def get_lms_ags_endpoint(location, pk=None, action=""):
```
Comment (giovannicimolin) [inline lti_consumer/utils.py:63]: @davidjoy @davestgermain This is basically replicating a DRF viewset with the readonly views here, and will be replaced by an actual ModelViewSet once the full LineItem Implementation is done.

**Comment by giovannicimolin** [comment]
This was the previous implementation, not flexible enough for expanding the full feature set of LTI AGS. Closing.

**Pull request closed without merging**